surveys/2024-annual-survey/translations/zh-tw.md

```diff
@@ -0,0 +1,909 @@
+### 您使用 Rust 嗎？
+
+Type: select one
+
+- 是的，我使用 Rust（用於任何目的，即使您只是在學習）
+- 不，我目前不使用 Rust，但我曾經使用過
+- 不，我從未使用過 Rust
+
+### 很遺憾聽到您已不再使用 Rust，是什麼原因使您停止使用它？
+
+Type: select all that apply
+
+- 缺少語言功能
+- 缺少套件
+- 缺少工具
+- 學習太難
+- 社群不太禮貌、不受歡迎或令人不快
+- 我更喜歡使用其他語言
+- 由於無法控制的因素，我不再有機會使用Rust
+- 如果將來有機會，我打算重新使用Rust
+- 其他
+
+### 請告訴我們更多原因：
+
+Type: free form
+
+### 您為什麼不使用 Rust？
+
+Type: select all that apply
+
+- Rust 不能幫助我達成目標
+- 缺少語言功能
+- 缺少套件
+- 缺少工具
+- 學習太難或者會花太多時間
+- 社群不太禮貌、不受歡迎或令人不快
+- 我更喜歡使用其他語言
+- 由於我無法控制的因素，我無法使用 Rust
+- 我沒有時間
+- 其他
+
+### 請告訴我們更多原因：
+
+Type: free form
+
+### 您使用 Rust 的平均頻率是多少？
+
+Type: select one
+
+- 每天或者幾乎每天
+- 每週或者幾乎每週
+- 每月或幾乎每月
+- 很少
+
+### 您如何評價自己的 Rust 開發能力？
+
+Type: select one
+
+- 我不會寫 Rust 程式碼
+- 我可以用 Rust 編寫簡單的程式
+- 我可以寫有用的、產品級的程式碼，但很艱難
+- 我寫 Rust 很有效率
+
+### 您是在什麽時候開始學習 Rust ？
+
+Type: select one
+
+- 我正在持續*積極地*學習 Rust。
+- 2024年期間
+- 2022年或2023年期間
+- 2020年或2021年期間
+- 2018年或2019年期間
+- 2017年或之前
+
+### 您從哪裡學習 Rust ？
+
+Type: select all that apply
+
+- 書籍（例如《The Rust 編程語言》、《Rust for Rustaceans》等）
+- 線上練習教學（例如 Rustlings、Rust by Examples 等）
+- 影片或直播
+- 部落格
+- 官方文件
+- crate 的原始碼
+- 線上課程、網路研討會
+- 個人教學
+- 大學教學資料
+- 其他（請注明）
+
+### 您在今年或去年是否參加過（或仍在參加）使用或教授 Rust 的課程或培訓？
+
+Type: select one
+
+- 是，通過大學、學校或其他教育機構
+- 是，通過我的雇主、承包商或諮詢公司
+- 否
+
+### 您經常使用哪些作業系統進行 Rust 開發?
+
+Type: select all that apply
+
+- Linux
+- Windows 10/11
+- Windows 8 或更早版本
+- 適用於 Linux 的 Windows 子系統（WSL）
+- macOS
+- 其他
+
+### 您經常以哪些作業系統作為目標系統進行 Rust 開發？
+
+Type: select all that apply
+
+- Linux（桌面或服務器）
+- Windows 10/11
+- Windows 8 或更早版本
+- macOS
+- iOS
+- 安卓（Android）
+- 嵌入式平台（帶作業系統）
+- 嵌入式平台（裸機）
+- WebAssembly（瀏覽器環境）
+- WebAssembly（非瀏覽器環境）
+- 明確平台獨立（如不與作業系統交互的套件）
+- 其他
+
+### 您通常使用什麽編輯器或 IDE 配置來編寫 Rust 代碼？
+
+Type: select all that apply
+
+- VS Code
+- vi、vim、neovim
+- IntelliJ、CLion、其他 JetBrains IDE + Rust 插件
+- Rust Rover（專用 IntelliJ Rust IDE）
+- Emacs（或 Doom Emacs、Spacemacs 等衍生品）
+- Sublime Text
+- Visual Studio
+- Xcode
+- Atom
+- Helix
+- 其他
+
+### 您在本地開發中使用什麽版本的 Rust？
+
+Type: select all that apply
+
+- 當前穩定版
+- 之前的穩定版
+- 不低於 1.75 的特定穩定版本
+- 低於 1.75 的特定穩定版本
+- Beta 版
+- 最新的 Nightly
+- 特定版本的 Nightly
+- 自定義的 fork
+- 我不知道
+- 其他
+
+### 如果您使用 Nightly 版，為什麽？
+
+Type: select all that apply
+
+- 我不使用 Nightly
+- 出於習慣
+- 為了我需要的特定語言功能
+- 我喜歡能用上所有最新功能
+- 幫助測試；檢測 Nightly 版本是否含有缺陷
+- 提供有關 Nightly 功能的設計反饋
+- 用於持續集成中測試
```

Comment:
Maybe we won't translate CI like this? 🤔 

```suggestion
- 用於 CI 中測試
```

Reply:
Thanks!